applications/DeepSpeed-Chat/training/utils/data/raw_datasets.py

```diff
@@ -11,7 +11,7 @@
 # follow in order to have a unified API and unified data format.
 class PromptRawDataset(object):
 
-    def __init__(self, output_path, seed, local_rank):
+    def __init__(self, output_path, seed, local_rank, dataset_name=None):
```

Comment:
dataset_name is a confusing variable name, I recommend to change it to local_data_dir. Same comment apply to all other classes.